packages/react-router/modules/RouterContext.js

```diff
@@ -1,9 +1,12 @@
 // TODO: Replace with React.createContext once we can assume React 16+
 import createContext from "create-react-context";
 
-const context = createContext();
-
-context.Provider.displayName = "Router.Provider";
-context.Consumer.displayName = "Router.Consumer";
+const createNamedContext = name => {
+  const context = createContext();
+  context.Provider.displayName = `${name}.Provider`;
+  context.Consumer.displayName = `${name}.Consumer`;
+  return context;
+}
 
+const context = /*#__PURE__*/ createNamedContext('Router');
```

Comment:
this could be added automatically with my https://github.com/Andarist/babel-plugin-annotate-pure-calls but I didnt want to incorporate a new dependency blindly

Reply:
Perfect, thanks 👍